<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="BHAVNAWKS506" failures="1" tests="3" name="Submit Order Test" time="58.169" errors="0" timestamp="2024-09-04T16:58:18 IST">
  <testcase classname="mortgageCreditLink.CloudQA_EndToEnd.SubmitOrder" name="SubmitOrderTest" time="16.388">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at mortgageCreditLink.CloudQA_EndToEnd.SubmitOrder.SubmitOrderTest(SubmitOrder.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- SubmitOrderTest -->
  <testcase classname="mortgageCreditLink.CloudQA_EndToEnd.SubmitOrder" name="SubmitOrderTest" time="16.564"/>
  <testcase classname="mortgageCreditLink.CloudQA_EndToEnd.SubmitOrder" name="OrderHistoryTest" time="0.0">
    <skipped/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order Test -->
